package-lock.json

```diff
@@ -5,15 +5,15 @@
   "requires": true,
   "dependencies": {
     "@types/mocha": {
-      "version": "5.2.5",
-      "resolved": "https://registry.npmjs.org/@types/mocha/-/mocha-5.2.5.tgz",
-      "integrity": "sha512-lAVp+Kj54ui/vLUFxsJTMtWvZraZxum3w3Nwkble2dNuV5VnPA+Mi2oGX9XYJAaIvZi3tn3cbjS/qcJXRb6Bww==",
+      "version": "2.2.48",
```

Comment:
What is going on here?

Reply:
(I know it's a dev dependency, but still seems like a weird version "increment")

Reply:
The package-lock.json works in mysterious ways...

Reply:
The inner machinations of npm are an enigma...